virtcontainers/container.go

```diff
@@ -603,6 +607,14 @@ func filterDevices(sandbox *Sandbox, c *Container, devices []ContainerDevice) (r
 			}).Info("Not attach device because it is a CDROM")
 			continue
 		}
+
+		if major == floppyMajor {
```

Comment:
as I think @jodh-intel implied - either now or if we add any more, we could fold the floppy and cd bits into a map/table and just have the one test loop maybe.